### PR TITLE
Add release notes for 0.264.1

### PR DESCRIPTION
**Pull request by gggrace14**

### CODE REVIEW

**Comment by highker** [comment]
If we really want to have this release, we will need something special for the release note. We haven't yet published any release note for materialized view so far. So having any release note related to it is weird. So my question would be: do we really need this release? Or if it is really necessary, let's maybe have a big materialized view release note for it. cc: @mayankgarg1990 @tdcmeehan

**Comment by gggrace14** [comment]
> If we really want to have this release, we will need something special for the release note. We haven't yet published any release note for materialized view so far. So having any release note related to it is weird. So my question would be: do we really need this release? Or if it is really necessary, let's maybe have a big materialized view release note for it. cc: @mayankgarg1990 @tdcmeehan

I'm more inclined to go for a big materialized view release note. My prior thought is to add the release note for introducing materialized view once we hit the milestone and have all basis/fundamental support in. And I think it is the time now.

Only concern is limited time. Oncall would like to minimize delay and deploy tomorrow. Is a few hours enough to iterate on a big release note?

File: presto-docs/src/main/sphinx/release/release-0.264.1.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.264.1
+===============
+
+General Changes
+_______________
+* Refine ACL check for expanded queries. (:pr:`16870`)
+* Add expanded query to keep track of the query expanded in StatementAnalyzer. (:pr:`16876`)
```
Comment (highker) [inline presto-docs/src/main/sphinx/release/release-0.264.1.rst:8]:
```
SPI Changes
_______________
Add ``expandedQuery`` field to ``QueryCompletedEvent`` to support query text logging with more information.
```

**Review by highker — CHANGES_REQUESTED** [review]

File: presto-docs/src/main/sphinx/release/release-0.264.1.rst (hunk 1 above)
Comment (gggrace14) [reply presto-docs/src/main/sphinx/release/release-0.264.1.rst]: Revised

File: presto-docs/src/main/sphinx/release/release-0.264.1.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+===============
+Release 0.264.1
+===============
+
+General Changes
+_______________
+* Refine ACL check for expanded queries. (:pr:`16870`)
```
Comment (highker) [inline presto-docs/src/main/sphinx/release/release-0.264.1.rst:7]: remove this

Comment (gggrace14) [reply presto-docs/src/main/sphinx/release/release-0.264.1.rst]: Done

**Review by highker — APPROVED** [review]

**Pull request merged**